.clang-tidy

```diff
@@ -36,7 +36,6 @@ cert-err34-c,
 cert-flp30-c,
 cppcoreguidelines-avoid-do-while,
 cppcoreguidelines-avoid-goto,
-cppcoreguidelines-avoid-non-const-global-variables,
```

Comment:
This rule appears to be made more for C++, so clang-tidy counts static C variables as global. Their argument for this is that translation units get too big commonly (if only there were a tool that would warn me if my files get too big), so even a variable only accessible in a single file is 'global'. I'll have to disable this rule for C projects in that case, since the alternatives don't exist in C.

Reply:
For reference: https://github.com/isocpp/CppCoreGuidelines/issues/2195